RecoLocalCalo/HcalRecAlgos/interface/PulseShapeFitOOTPileupCorrection.h

```diff
@@ -42,16 +41,20 @@ class PulseShapeFitOOTPileupCorrection
 		     const std::vector<double> & its4Chi2, HcalTimeSlew::BiasSetting slewFlavor, int iFitTimes);
 
     const HcalPulseShapes::Shape* currentPulseShape_=nullptr;
+    const HcalTimeSlew* hcalTimeSlewDelay_=nullptr;
+    double TSdelay1GeV_=0;
```

Comment:
start local variables and data members with lower case letter. Capitalization is reserved for class names.